### DISCUSSION TITLE
Make div inside card fill available height

### DISCUSSION THREAD

**Discussion by machinchose** [quasarframework/quasar#10783] [General - Components / Directives / etc] [answered]
Hello everyone,

I am trying to make a loading menu for a personal project (beginner) and I am struggling with the following question : 
how can I make the content of a card fill the card but not overflow. 
I have the following pen https://codepen.io/elazard/pen/ExXRXgg which shows what I am trying to achieve. my issue with that is that I fixed the height to some vh value but I think there is probably a way to achieve the same result with something more dynamic that would be screen size agnostic. 
if I understand correctly the answer to my issue should be flexbox but I played around with it (full-height class and so on) but I cannot make it work. the best I achieved, it would take the full size of the card not just the available height so I had an overflow the size of the Load title div. 
A little nudge in the right direction would be really appreciated 
thank you very much  !

**Comment by pikil** [comment, marked as answer]
The problem is ```height: 55vh``` for scroll-area

Add ```column``` class to that ```<q-card-section />``` and apply ```col``` on ```<q-scroll-area />``` - that should do the trick

```
...
          <q-card-section class="col-9 column">
             <div class="col">
               <q-scroll-area class="full-height">
                 <span v-for="m in 20" :key="m">
                   Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum
                   repellendus sit voluptate voluptas eveniet porro. Rerum blanditiis
                   perferendis totam, ea at omnis vel numquam exercitationem aut, natus
                   minima, porro labore.
                 </span>
               </q-scroll-area>
             </div>
             <div class="col-auto">
               <q-card-actions>
                 <q-btn flat label="OK" color="primary" v-close-popup />
               </q-card-actions>
             </div>
          </div>
...
```

**Reply by machinchose** [reply]
hey, thank you for your answer. I tried to apply what you said and it worked great until I also tried to apply it to the other "side" of the card (the one with the list) https://codepen.io/elazard/pen/bGRxRRj now it's not full size anymore. what am I missing? thank you  !

**Reply by pikil** [reply]
q-card-sections should be the ones containing `col` classes, but you enclose them in div

```
    <q-card flat bordered style="height: 80vh" class="column">
        <q-card-section class="text-h4 text-primary q-pb-xs col-auto">
            Load
        </q-card-section>
        <q-card-section class="text-h4 text-primary q-pb-xs col-auto">
            ...

     </q-card>

**Reply by machinchose** [reply]
I think I am starting to see the line, thanks a lot !
